bin/pbiviz.js

```diff
@@ -195,8 +204,33 @@ function createCertFile() {
     }
 }
 
+function getCertFile(config) {
+    let  cert = path.join(__dirname, '..', config.server.certificate);
+    let  pfx = path.join(__dirname, '..', config.server.pfx);
+
+    if (!fs.existsSync(cert) && !fs.existsSync(pfx)) {
```

Comment:
Looks like the same condition are double checked below.
Please remove this condition and move the following code to the end of this function:
```typescript
ConsoleWriter.info('Certificate not found. Call `pbiviz --create-cert` command to create the new certificate');
return null;
```